docs/api/index.rst

```diff
@@ -27,3 +27,4 @@ For those of you who want the gritty details.
     commands
     main
     app
+    interactive
```

Comment:
shouldn't dates and the other new files be added here?

Reply:
Is there a nice command for that or should I write everything by hand?

Reply:
I think you need to do it by hand?